functions/src/admin/AdminPaymentTaskProcessor.ts

```diff
@@ -77,6 +81,64 @@ export class AdminPaymentTaskProcessor {
 		return `Created ${paymentsCreated} payments`;
 	};
 
+	private sendNotifications = async (recipientDocs: QueryDocumentSnapshot<Recipient>[]) => {
+		let newNotificationsSent = 0;
+		let existingNotifications = 0;
+		const now = moment();
+
+		for (const recipientDoc of recipientDocs) {
+			if (recipientDoc.data().test_recipient) continue;
+
+			const paymentDocRef = this.firestoreAdmin.doc<Payment>(
+				`${RECIPIENT_FIRESTORE_PATH}/${recipientDoc.id}/${PAYMENT_FIRESTORE_PATH}`,
+				now.format('YYYY-MM')
+			);
+
+			if ((await paymentDocRef.get()).exists) {
+				const paymentDocSnap = await paymentDocRef.get();
+				const payment: Payment = paymentDocSnap.data() as Payment;
+				if (!payment.message) {
+					const recipient: Recipient = recipientDoc.data();
+					const [messageSid, _messageStatus, messageContent] = await sendSms({
+						messageRecipientPhone: '+' + recipient.mobile_money_phone.phone,
+						messageContext: {
+							content: 'This is a Test SMS',
+						},
+						smsServiceId: TWILIO_SID,
+						smsServiceSecret: TWILIO_TOKEN,
+						messageSenderPhone: TWILIO_SENDER_PHONE,
+						templateParameter: {
+							language: 'de',
```

Comment:
Dynamically, based on the recipient language?

Reply:
as it is freetext, I think the language is irrelevant for that specific case :-)